certora/harnesses/GuardianStorageHarness.sol

```diff
@@ -27,4 +27,20 @@ contract GuardianStorageHarness is GuardianStorage {
             count++;
         }
     }
+
+    /**
+     * @notice Gets the list of guardians and check if the passed address is a guardian in that list.
+     * @param _wallet The target wallet.
+     * @param _checkGuardian The guardian to check.
+     * @return _checkGuardian is guardian or not.
+     */
+    function isGuardianInGuardianList(address _wallet, address _checkGuardian) public view returns (bool) {
+        address[] memory guardians = getGuardians(_wallet);
```

Comment:
Why can't you write this in CVL? You already had this nice "trick" with the `index` in the other PR that can be used here (unless I'm missing something).

Reply:
Done 👍🏾